lib/SILOptimizer/Utils/Local.cpp

```diff
@@ -1607,10 +1623,10 @@ optimizeBridgedSwiftToObjCCast(SILInstruction *Inst,
 
   auto &M = Inst->getModule();
   auto Loc = Inst->getLoc();
-  
+
+  bool AddressOnlyType = false;
   if (!Src->getType().isLoadable(M) || !Dest->getType().isLoadable(M)) {
-    // TODO: Handle address-only types.
-    return nullptr;
+    AddressOnlyType = true;
```

Comment:
I see. Paranoia is well and good, but I think the SIL verifier, if not something else, would adequately catch these particular inconsistencies. I'll leave it up to you whether to leave the check here or not (it'll at least optimize away in no-asserts builds).